apollo-environment/pom.xml

```diff
@@ -76,6 +85,19 @@
             <artifactId>hamcrest-library</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- for com.spotify.apollo.meta.model.benchmark.GathererBenchmark -->
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-core</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-generator-annprocess</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
They are there, but I forgot to remove them from apollo-environment - now done, thanks!